### PR TITLE
Pull docker containers in the background

### PR DESCRIPTION
**Pull request by SerialVelocity**
Based off Himangi's branch

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/1330)

### CODE REVIEW

File: scripts/circle-ci/common-containers.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cassandra2:
+  image: palantirtechnologies/docker-cassandra-atlasdb:2.2.8
+
+#cassandra3: //Uncomment when Cassanra 3.7 testing is enabled.
```
Comment (jeremyk-91) [inline scripts/circle-ci/common-containers.yml]: nit: `Cassandra`

File: scripts/circle-ci/common-containers.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+postgres:
+  image: kiasaki/alpine-postgres
+
+nginx:
+  image: 1science/nginx
+
+socks:
+  image: vimagick/dante:latest
+
+jepson:
```
Comment (jeremyk-91) [inline scripts/circle-ci/common-containers.yml]: `jepsen` - we don't reference the keys so I guess this is probably fine, but let's try to keep this clear

**Comment by jeremyk-91** [comment]
👍

File: scripts/circle-ci/common-containers.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+postgres:
+  image: kiasaki/alpine-postgres
+
+nginx:
+  image: 1science/nginx
+
+socks:
+  image: vimagick/dante:latest
+
+jepsen:
```
Comment (harrybiddle) [inline scripts/circle-ci/common-containers.yml]: No need for Jepsen here: we're not actually using it on CircleCI

Comment (harrybiddle) [inline scripts/circle-ci/common-containers.yml]: Realise I'm too late with this comment but ah, I'm saying it anyway

**Pull request merged**